src/terminal/kitty/key.zig

```diff
@@ -83,6 +83,15 @@ pub const Flags = packed struct(u5) {
     report_all: bool = false,
     report_associated: bool = false,
 
+    /// Sets all modes on.
+    pub const @"true": Flags = .{
```

Comment:
I didn't end up using this in this PR (I tried for a bit) but this is useful to have so I kept it around.